Dockerfile

```diff
@@ -25,7 +25,7 @@
 # such as `--build-arg TOMCAT_JRE=jre8-alpine`
 #
 ARG TOMCAT_VERSION=8.5
-ARG TOMCAT_JRE=jre8
+ARG TOMCAT_JRE=jdk8
```

Comment:
Why?

Reply:
As discussed above with @manolan1, and also based on his comment in the [issue itself](https://issues.apache.org/jira/browse/GUACAMOLE-890), this is a more uptodate / supported Tomcat image, which also helps with non-root user, as it contains some of the fixes we need (especially the permissions on the `temp` folder).

Reply:
Hmmm...okay...this is going to increase the size of this Docker image pretty substantially, I would imagine, so I'm a little skeptical that this is the right thing to do.  Sorry I missed it in the other conversation...

Reply:
I also don't think a world-writable directory is the right way to go - the directory should have the correct ownership and permissions, not just the ones that work because we've blown everything open.  If we're trying to improve security with this issue, making something world-writable seems contradictory to that effort.

Reply:
I've then reverted to original `jre` image, and created a user-writable `temp` dir.
This was what I initially pushed before switching to `sdk` image.